README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+A. While presentational HTML makes your webpage look good, semantic is used for the actual information you want your page to display.
```

Comment:
Good answers here, another thing to keep in mind is the  search engines and screen readers will have more info about your content and it's meaning as well